website/vercel.json

```diff
@@ -1201,6 +1201,46 @@
     {
       "source": "/getting-started/tutorials/customer-support-tool/application-settings",
       "destination": "/getting-started/tutorials/start-building"
+    },
+    {
+      "source": "/widget-reference/file-picker",
```

Comment:
This is a wrong URL, can we please add it for deindexing.

Reply:
deindexed